flake.nix

```diff
@@ -196,49 +198,57 @@
           evm-assigner-gcc = (pkgs.callPackage ./evm-assigner/evm-assigner.nix {
             runTests = true;
             enableDebug = false;
-            crypto3 = packages.crypto3;
+            crypto3 = packages.parallel-crypto3;
```

Comment:
why used from packages but not the parallel-crypto3-gcc?

Reply:
That made on purpose to:

test cmake configuration without tests. I had a case, when I made it invalud without noticing it
reach higher level of parallelizm. Nix can start compiling evm-assigner and proof-producer simultaniously

Reply:
so why another checks (like parallel-crypto3-clang) doesn't use crypto3 from package.
My concern is: some checks used crypto3 rom package another - not, why?, what different between them?

Reply:
It's simple, because there is no fitting clang derivation. There are already too many.
I could do it. But it's just copy-paste. And clang is faster in the CI anyway